ocs_ci/ocs/amq.py

```
@@ -306,9 +308,9 @@ def setup_amq_kafka_bridge(self):
                 os.path.join(self.dir, self.amq_kafka_bridge_yaml)
             )
             self.kafka_bridge = OCS(**kafka_bridge)
-            self.kafka_bridge.create()
+            self.kafka_bridge.create(out_yaml_format=False, log_yaml=False)
         except (CommandFailed, CalledProcessError) as cf:
-            log.error("Failed during setup of AMQ KafkaConnect")
+            log.error("Failed during setup of AMQ KafkaBridge")
```

Ruff (0.15.21)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)